test/test_rosdep_main.py

```diff
@@ -280,6 +280,34 @@ def test_keys(self):
         except SystemExit:
             pass
 
+    def test_search(self):
+        sources_cache = get_cache_dir()
+        cmd_extras = ['-c', sources_cache]
+
+        try:
+            with fakeout() as b:
+                rosdep_main(['search', 'curl'] + cmd_extras)
+                stdout, stderr = b
+                assert 'Closest keys' in stdout.getvalue(), stdout.getvalue()
+                assert 'curl' in stdout.getvalue(), stdout.getvalue()
+                assert 'Closest packages' not in stdout.getvalue(), stdout.getvalue()
+                assert not stderr.getvalue(), stderr.getvalue()
+            with fakeout() as b:
+                rosdep_main(['search', 'libeigen3-dev'] + cmd_extras)
+                stdout, stderr = b
+                assert 'Closest keys' not in stdout.getvalue(), stdout.getvalue()
+                assert 'eigen:' in stdout.getvalue(), stdout.getvalue()
+                assert 'Closest packages' in stdout.getvalue(), stdout.getvalue()
```

Comment:
I'd just move the `eigen:` line down so it's under `'Closest packages'`.

```suggestion
                assert 'Closest keys' not in stdout.getvalue(), stdout.getvalue()
                assert 'Closest packages' in stdout.getvalue(), stdout.getvalue()
                assert 'eigen:' in stdout.getvalue(), stdout.getvalue()
```